homeassistant/components/config/__init__.py

```diff
@@ -23,6 +23,11 @@ def async_setup(hass, config):
     yield from hass.components.frontend.async_register_built_in_panel(
         'config', 'config', 'mdi:settings')
 
+    # Temporary way of allowing people to opt-in for config-entries
+    if config.get(DOMAIN, {}).get('config_entries'):
+        global SECTIONS
+        SECTIONS = SECTIONS + ('config_entries',)
```

Comment:
I think we should do it like ON_DEMAND

Reply:
Looks like `ON_DEMAND` is a string not a tuple. Is that a bug?

Reply:
that's a bug. However, since we check with `in`, it actually worked, oops. will fix.